lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -418,46 +405,43 @@ def calculate_wet_bulb_temperature(self, temperature, relative_humidity,
         precision = np.full(temperature.data.shape, self.precision)
 
         # Set units of input diagnostics.
-        relative_humidity.convert_units(1)
-        pressure.convert_units('Pa')
```

Comment:
These functions were ensuring that the data is in the right units. How can we be sure the incoming data is correct?